web/settings/modulconfigbat.php

```diff
@@ -335,16 +335,29 @@ function visibility_solarwatt_ip2() {
 						<div id="divspeicheralphaess" class="hide">
 							<div class="form-group">
 								<div class="form-row mb-1">
-									<label for="vartaspeicherip" class="col-md-4 col-form-label">Firmware Version</label>
+									<label for="speicheralphaess" class="col-md-4 col-form-label">Firmware Version</label>
 									<div class="col">
 										<select name="alphav123" id="alphav123" class="form-control">
-											<option <?php if($alphav123old == "0") echo "selected" ?> value="0">EMS älter als 1.23V</option>
-											<option <?php if($alphav123old == "1") echo "selected" ?> value="1">EMS 1.23V oder neuer</option>
+											<option <?php if($alphav123old == "0") echo "selected" ?> value="0">Alpha ESS-Kit: EMS älter als 1.23V</option>
+											<option <?php if($alphav123old == "1") echo "selected" ?> value="1">Alpha ESS-Kit: EMS 1.23V oder neuer</option>
+											<option <?php if($alphav123old == "2") echo "selected" ?> value="2">Hi5/10</option>
```

Comment:
Wie schon oben im Code geschrieben, sollte das in eine separate Einstellung ausgelagert werden.